doc/releases/changelog-dev.md

```diff
@@ -235,6 +235,11 @@
 
 <h3>Breaking changes 💔</h3>
 
+* The functions `qml.transforms.metric_tensor`, `qml.transforms.adjoint_metric_tensor` and  
+  `qml.transforms.classical_jacobian` are moved to the gradients module `qml.gradients` and therefore become
+  `qml.gradients.metric_tensor`, `qml.gradients.adjoint_metric_tensor` and `qml.gradients.classical_jacobian`.
+  [(#4900)](https://github.com/PennyLaneAI/pennylane/pull/4900)
```

Comment:
> Though @josh146 I guess you're thinking that it should be rare for users to have not used the top-level option?

@trbromley more so the way I see it: how we publicly document PennyLane is the user facing API. If we, in all of our examples and documentation, direct users to use `qml.metric_tensor`, than this is the user-facing interface that we need to avoid breaking.

But its actual source location (`qml.transforms.metric_tensor`) is an implementation detail, the same way that `qml.ops.qubit.qchem_ops.SingleExcitation` is an implementation detail.

If we were to move `SingleExcitation` to `qml.ops.qubit.qchem.SingleExcitation` for example, I wouldn't advertise this as a breaking change. Sure, some users might have found this and used it from this location, but we never documented this as the API to access the `SingleExcitation` op, and never guaranteed stability for this import path.

Reply:
Thanks, sounds good!

Reply:
Ok thanks, I will remove it from the breaking change then!